invenio_search/ext.py

```diff
@@ -405,6 +405,20 @@ def _build(tree_or_filename, alias=None):
                     name=action["alias"],
                     ignore=ignore,
                 )
+    def update_mapping(self, index):
```

Comment:
```suggestion

    def update_mapping(self, index):
```

nit: needs a newline above :)